source/material_model/melt_phipps_morgan.cc

```diff
@@ -0,0 +1,1363 @@
+                             "and reference pressure."
+                             "Units: Pa.");
+          // Holland & Powell Table 2a
+          prm.declare_entry ("First derivatives of the bulk modulus", "4.68, 3.84, 10.42, 10.06",
+                             Patterns::List(Patterns::Double()),
+                             "The pressure derivative of the bulk modulus at the reference temperature and reference "
+                             "pressure for each different endmember component."
+                             "Units: none.");
+          // Holland & Powell Table 2a, assuming they are given in [1/kbar = 1/(1e8 Pa)] Q!
+          prm.declare_entry ("Second derivatives of the bulk modulus", "-3.7e-11, -3.0e-11, -3.59e-10, -2.78e-10",
+                             Patterns::List(Patterns::Double()),
+                             "The second pressure derivative of the bulk modulus at the reference temperature and reference "
+                             "pressure for each different endmember component."
+                             "Units: 1/Pa.");
+          // Q!
+          prm.declare_entry ("Einstein temperatures", "418.1, 561.0, 505.75, 558.1 ",
+                             Patterns::List(Patterns::Double(0)),
+                             "List of Einstein temperatures for each different endmember."
+                             "Units: K.");
+          // Holland & Powell Table 2a, \Delta f H, given in kJ/mol Q!
+          prm.declare_entry ("Reference enthalpies", "-1477740.0, -2172570.0, -1463040.0, -2237320.0",
+                             Patterns::List(Patterns::Double()),
+                             "List of enthalpies at the reference temperature and reference "
+                             "pressure for each different endmember component."
+                             "Units: J/mol.");
+          // Holland & Powell Table 2a, S, given in [J/K/mol]
+          prm.declare_entry ("Reference entropies", "151.0, 95.1, 96.0, -62.0",
+                             Patterns::List(Patterns::Double()),
+                             "List of entropies at the reference temperature and reference "
+                             "pressure for each different endmember component."
+                             "Units: J/K/mol.");
+          // Holland & Powell Table 2a, a, given in [kJ/K/mol]
+          prm.declare_entry ("Reference specific heat capacities", "201.1, 233.3, 243.7, 269.4",
+                             Patterns::List(Patterns::Double(0)),
+                             "List of specific heat capacities for each different endmember at the reference temperature "
+                             "and reference pressure."
+                             "Units: J/kg/K.");
+          // Holland & Powell Table 2a, b, given in [1e5 kJ/K^2/mol] (so *1e3/1e5) Q!
+          // Q! in D2021, Table 1A, the unit is J/K2/mol instead of J/kg/K/K.
```

Comment:
Yes, these are molar values, rather than specific values - thermodynamics is easier in molar quantities. We convert to specific values before using the heat capacities in the heat equation.

Reply:
It's just that the unit given for C1 in `declare_parameters` of melt_boukare is J/kg/K/K and in table A1 J/K2/mol, while the values for C1 in both places are the same. Same for C2. I'll change the units here to molar.